### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ExtractDacpac_ExtractDacpacSettings_Keyboard:The first focus is present on the next button.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6846]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version:1.10.0-insider (user setup)
Commit: 1ff9a2e
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tools: Voiceover
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select second radio button.
9. Navigate till next button & hit enter.
10. Start screen reader as voiceover and navigate to the version edit box.
11. Clear the default value.

**Actual:**
Navigate to the version edit box and clear the value, the error message gets displayed screen reader does not announce it.

**Same issue is observed for the following page:**
1.Import bacpac setting:
Follow the exact above steps but select the third radio button
Issue observed for "File location" and "Target database".

**Expected:**
The screen reader should announce the error message as soon as it pop-up as well as when navigate to it.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not receive the information about the error message

**MAS Reference:**
MAS1.3.1:[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**

**Comment by udeeshagautam** [comment]
@fsteffi I am not able to repro the above the NVDA reader is reading the error and showing in speech viewer too. Should I be using some other screen reader for this. Can you please help me with the repro.

![image](https://user-images.githubusercontent.com/46980425/63461773-418f3000-c40e-11e9-9d5b-cd102af547de.png)

**Comment by udeeshagautam** [comment]
NVDA seems to work. On hold with reader specific issues.

**Comment by fsteffi** [comment]
@udeeshagautam this bug is voiceover specific.

**Comment by ellbosch** [comment]
For Mac, I can confirm that extracting a Dacpac results in correct functionality for VoiceOver.

However, Mac VoiceOver does not work as expected for importing Bacpac. Required fields are instantiated as empty strings, and VoiceOver does not read the displayed errors when the view opens.

**Comment by ellbosch** [comment]
Follow-up—VS Code does not appear to suffer from these issues. I'll take a look at our code.
